inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/iceberg/IcebergSourceOperator.java

```diff
@@ -83,4 +96,52 @@ public StreamSource getFromEntity(StreamSourceEntity entity) {
         source.setFieldList(sourceFields);
         return source;
     }
+
+    @Override
+    @Transactional(rollbackFor = Throwable.class, isolation = Isolation.REPEATABLE_READ)
+    public void getFieldInfo(SourceRequest request, String operator) {
```

Comment:
the method name is get field info but the actual action is update iceberg field info

Reply:
Fixed.